IBF-Typhoon-model/Dockerfile

```diff
@@ -115,12 +100,23 @@ RUN apt-get install -y python3-eccodes
 COPY requirements.txt /home/fbf/
 RUN python3.7 -m pip install --no-cache-dir -r requirements.txt
 
+COPY data /home/fbf/data
+COPY data-raw /home/fbf/data-raw
+COPY lib /home/fbf/lib
+COPY lib_r /home/fbf/lib_r
+COPY logos /home/fbf/logos
+COPY lib_r /home/fbf/lib_r
+COPY models /home/fbf/models
+COPY mainpipeline.py /home/fbf/
+COPY setup.py /home/fbf/
+COPY run_model_V2.R /home/fbf/
+COPY __init__.py /home/fbf/
 # set up cronjob
 # COPY crontab /etc/cron.d/crontab
 # RUN chmod 0644 /etc/cron.d/crontab
 # RUN crontab /etc/cron.d/crontab
 # RUN touch /var/log/cron.log
 # CMD cron && tail -f /var/log/cron.log
-
+RUN pip install .
```

Comment:
This won't work unless all the files are copied in, so if you do remove the above lines you will have to run this only after spinning up the docker container